src/java.base/share/classes/java/lang/reflect/code/interpreter/Interpreter.java

```diff
@@ -518,6 +599,19 @@ static Object exec(MethodHandles.Lookup l, OpContext oc, Op o) {
             MethodHandle mh = opHandle(o.opName() + "_" + o.opType().returnType(), o.opType());
             Object[] values = o.operands().stream().map(oc::getValue).toArray();
             return invoke(mh, values);
+        } else if (o instanceof CoreOps.AssertOp _assert) {
+            Body testBody = _assert.bodies.get(0);
+            Boolean testResult = (Boolean) invokeBody(l,testBody,oc);
```

Comment:
Consistent style and can cast to primitive type:

```suggestion
            boolean testResult = (boolean) invokeBody(l, testBody, oc);
```